### ISSUE TITLE
Replace DDB ID with gene name in extensions

### ISSUE THREAD

**Issue by wildlifehexagon** [dictyBase/Genomepage#49]
Locally this can be accessed via `https://betafunc.dictybase.local/goa/converter/{gene_id}`.

See https://github.com/dictybase-playground/kubeless-nodefn/tree/master/geneids

**Comment by wildlifehexagon** [comment]
I'm running into issues adding this into the backend. I made some notes inside [feature/geneid2name](https://github.com/dictybase-playground/kubeless-nodefn/tree/feature/geneid2name), but my problem is that when I try converting extensions using my `convertExtensions` function, I always get an empty object as a response.

This is related to the same issue we ran into before involving using `async/await` with `.map`. What's getting tripped up this time is likely the use of nested arrays. I have tried explicitly returning a `Promise.all` on the map output, but no matter what it's sending back the empty object.

Due to these issues, it may be preferable to put the logic into the front end for now.
